data/actionsmap/yunohost.yml

```diff
@@ -226,6 +226,97 @@ user:
                 username:
                     help: Username or email to get information
 
+#############################
+#           Alias            #
+#############################
+alias:
+    category_help: Manage alias
+    actions:
+
+        ### alias_list()
+        list:
+            action_help: List existing aliases
+            api: GET /alias
+            configuration:
+                authenticate: all
+                authenticator: ldap-anonymous
```

Comment:
First time this function is called (by `yunohost alias list`), even after a `regen-conf` of slapd, the anonymous ldap connection breaks `_ensure_ldap_ou_is_created`.
See this log: https://paste.yunohost.org/cusucofise.vala